### ISSUE TITLE
Should there be an `Alias` class?

### ISSUE THREAD

**Issue by kwinkunks** [agile-geoscience/welly#117]
Things like #74 and maybe the things discussed in #114 might be easier if `alias` was a class and not just a `dict`.

**Comment by kinverarity1** [comment]
Yes please 😊 the ability to load a custom list would be very useful!

**Comment by kwinkunks** [comment]
What might that list look like? CSV type of thing?

**Comment by kinverarity1** [comment]
Yes, initially a CSV or pandas DF would be fine, mapping a range of different mnemonics (my datasets have really odd and contradictory mnemonics to Crains list) to a curve type. Is that what ``alias`` is aimed at?

**Comment by AlexC84** [comment]
Are you aware of this: [PWLS Version 3.0](https://www.energistics.org/pwls-version-3-0-candidate-release-available-for-review/)
